irr/src/CIrrDeviceSDL.h

```diff
@@ -322,28 +327,13 @@ class CIrrDeviceSDL : public CIrrDeviceStub
 
 	core::rect<s32> lastElemPos;
 
-	struct SKeyMap
-	{
-		SKeyMap() {}
-		SKeyMap(s32 x11, s32 win32) :
-				SDLKey(x11), Win32Key(win32)
-		{
-		}
-
-		s32 SDLKey;
-		s32 Win32Key;
-
-		bool operator<(const SKeyMap &o) const
-		{
-			return SDLKey < o.SDLKey;
-		}
-	};
-
-	core::array<SKeyMap> KeyMap;
+	std::unordered_map<SDL_Keycode, EKEY_CODE> KeyMap;
 	SDL_SysWMinfo Info;
 
 	s32 CurrentTouchCount;
 	bool IsInBackground;
+
+	std::unordered_set<SDL_Scancode> escapeKeys;
```

Comment:
fwiw considering this has a very low amount of elements a `std::set` would be more efficient

Reply:
https://github.com/luanti-org/luanti/pull/14964/commits/b7c6e3ec8b6453ae0a1ef058f119e7ee55d37795